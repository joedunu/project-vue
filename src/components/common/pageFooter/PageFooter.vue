<template>
    <div class='page-footer'>

    </div>
</template>

<script>
export default {
  name: "page-footer"
};
</script>

<style scoped>
.page-footer {
  height: 200px;
  background-color: #f19f4d;
}
</style>
