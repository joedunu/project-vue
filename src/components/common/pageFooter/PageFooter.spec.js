import { shallow } from "@vue/test-utils"
import PageFooter from "./PageFooter.vue"

describe("PageFooter", () => {
	test("snapshot: PageFooter", () => {
		let cmp = shallow(PageFooter, { data: {} })
		expect(cmp.element).toMatchSnapshot()
	})
})
