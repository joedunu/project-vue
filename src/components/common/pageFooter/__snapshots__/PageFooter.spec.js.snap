// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageFooter snapshot: PageFooter 1`] = `
<div
  class="page-footer"
/>
`;
